my_project/template_graphs.py

```diff
@@ -157,7 +161,7 @@ def yearly_profile(df, var, global_local):
             marker_color="silver",
             marker_opacity=0.3,
             hovertemplate=(
-                "Max: %{y:.2f} &#8451;<br>" + "Min: %{base:.2f} &#8451;<br>"
+                "Max: %{y:.2f} " + var_unit + "Min: %{base:.2f} " + var_unit
             ),
```

Comment:
The unit for 'adaptive comfort 80%' and 'adaptive comfort 90%' is defined statically, and I changed it to refer to the mapping_dictionary.